src/Compilers/VisualBasic/Portable/Lowering/UseTwiceRewriter.vb

```diff
@@ -321,6 +322,42 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 receiver = New Result(New BoundSequence(capture.Syntax, ImmutableArray(Of LocalSymbol).Empty, ImmutableArray.Create(Of BoundExpression)(capture), boundTemp, boundTemp.Type),
                                       boundTemp)
+            ElseIf receiverOpt.IsLValue AndAlso
+                   CodeGenerator.IsPossibleReferenceTypeReceiverOfConstrainedCall(receiverOpt) AndAlso
+                   Not CodeGenerator.ReceiverIsKnownToReferToTempIfReferenceType(receiverOpt) AndAlso
+                   Not (isForRegularCompoundAssignment AndAlso CodeGenerator.IsSafeToDereferenceReceiverRefAfterEvaluatingArguments(node.Arguments)) Then
+
+                Debug.Assert(Not receiverOpt.Type.IsReferenceType)
+
+                ' A case where T Is actually a class must be handled specially.
+                ' Taking a reference to a class instance Is fragile because the value behind the 
```

Comment:
```suggestion
                ' A case where T is actually a class must be handled specially.
                ' Taking a reference to a class instance is fragile because the value behind the 
```